### PR TITLE
Update deprecated node installation method

### PR DESCRIPTION
**Pull request by lamoglia**
Following instructions from https://github.com/nodesource/distributions

supersedes #11

### CODE REVIEW

**Comment by Remi-Gau** [comment]
pushed commits after fixing a small double quote bug in the make file and generating the dockerfile

the rest is linting and making sure that the circle-CI would be run on the PR

**Comment by Remi-Gau** [comment]
also updated to use the latest version of the validator

**Comment by Remi-Gau** [comment]
okidoki it can be built: merging.

thanks @lamoglia

File: Dockerfile [hunk 1]
```diff
@@ -3,14 +3,18 @@
...
 RUN apt-get update -qq \
            && apt-get install -y -q --no-install-recommends \
-                  ca-certificates curl apt-utils \
+                  ca-certificates curl apt-utils gnupg \
            && rm -rf /var/lib/apt/lists/*
+RUN mkdir -p /etc/apt/keyrings
+RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+ENV NODE_MAJOR="18"
+RUN echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | tee /etc/apt/sources.list.d/nodesource.list
 RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
```
Comment (Remi-Gau) [inline Dockerfile:12]: I think this line should not be there

Comment (lamoglia) [reply Dockerfile:12]: you're right, that was to be removed as proposed here https://github.com/bids-apps/base_validator/pull/11/files

Comment (lamoglia) [reply Dockerfile:12]: https://github.com/bids-apps/base_validator/pull/13

**Pull request merged**